cosmos/evmos_9001.json

```diff
@@ -35,6 +35,12 @@
       "coinDecimals": 18,
       "coinGeckoId": "evmos",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/evmos_9001/aevmos.png"
+    },
+    {
+      "coinDenom": "CROWDP",
+      "coinMinimalDenom": "erc20/0xfbF4318d24a93753F11d365A6dcF8b830e98Ab0F",
+      "coinDecimals": 18,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/evmos_9001/crowdp.png"
```

Comment:
And the image directory should follow `coinMinimalDenom`. Please take a look at the README and other chain informations.

Reply:
It has to be `erc20/0xfbF4318d24a93753F11d365A6dcF8b830e98Ab0F`

Reply:
`coinMinimalDenom` is `erc20/0xfbF4318d24a93753F11d365A6dcF8b830e98Ab0F` now
i think that is same with what you want

Reply:
meant the image directory

Reply:
i am really sorry but i am not sure what you mean
would you give me suggested code?

Reply:
i made some change
are you okay with that?

Reply:
![image](https://github.com/user-attachments/assets/5f3dfe3a-e982-48bb-8c77-a7f0030c292b)
like this?